cookiecutter-django-ida/{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+
+**Merge checklist:**
+Check off if complete *or* not applicable:
+- [ ] Documentation updated (not only docstrings)
+- [ ] Commits are squashed
+- [ ] Unit tests added/updated
+- [ ] Manual testing instructions provided
+- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets
+
+**Post-merge:**
+- [ ] Backported to [latest and next named releases](https://docs.openedx.org/en/latest/community/release_notes/named_release_branches_and_tags.html#latest-open-edx-release)
```

Comment:
```suggestion
- [ ] [Backported](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2065367719/Making+a+pull+request+for+a+named+release) to latest and next named releases
```
I think a link to the "How to backport" wiki page will be more useful. BTR can also update that page to include:
* the [link to latest named release](https://docs.openedx.org/en/latest/community/release_notes/named_release_branches_and_tags.html#latest-open-edx-release)
* how to tell if the next named release is cut
* how to decide whether a backport is necessary